tests/run-bank.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# TODO: update existing script.
+# Temporary since a custom oasis-node version is needed.
+
+set -o nounset -o pipefail -o errexit
+trap "exit 1" INT
+
+# Oasis Core release version to test against.
+OASIS_RELEASE=21.0.1
+
+# Get the root directory of the tests dir inside the repository.
+ROOT="$(cd $(dirname $0); pwd -P)"
+
+# What to use for GNU tar.
+TAR=tar
+
+# ANSI escape codes to brighten up the output.
+RED=$'\e[31;1m'
+GRN=$'\e[32;1m'
+CYAN=$'\e[36;1m'
+OFF=$'\e[0m'
+
+# The base directory for all the node and test env cruft.
+TEST_BASE_DIR=$(mktemp -d -t oasis-sdk-e2e-XXXXXXXXXX)
+
+# Kill all dangling processes on exit.
+cleanup() {
+	printf "${OFF}"
+	pkill -P $$ || true
+	wait || true
+}
+trap "cleanup" EXIT
+
+# Find build tools.
+if [[ "$(which go)" == "" ]]; then
+	printf "${RED}### Please install 'go'.${OFF}\n"
+	exit 1
+fi
+if [[ "$(which cargo)" == "" ]]; then
+	printf "${RED}### Please install 'cargo'.${OFF}\n"
+	exit 1
+fi
+
+# Find a downloader tool.
+if [[ "$(which wget)" == "" ]]; then
+	if [[ "$(which curl)" == "" ]]; then
+		printf "${RED}### Please install 'wget' or 'curl'.${OFF}\n"
+		exit 1
+	else
+		DOWNLOAD="curl --progress-bar --location -o"
+	fi
+else
+	DOWNLOAD="wget --quiet --show-progress --progress=bar:force:noscroll -O"
+fi
+
+printf "${CYAN}### Building test simple-bank runtime...${OFF}\n"
+cd "${ROOT}"/runtimes/simple-bank
+cargo build
+cp "${ROOT}"/../target/debug/test-runtime-simple-bank "${TEST_BASE_DIR}"/
+
```

Comment:
```suggestion
print_status() { printf "${CYAN}### $1...${OFF}\n"; }
print_status "Building e2e test harness"
```
and so forth. separates style from content